Files/Interacts/Interaction.cs

```diff
@@ -1114,7 +1114,7 @@ public void PushJumpChar(char letter)
             App.CurrentInstance.ViewModel.JumpString += letter.ToString().ToLower();
         }
 
-        public async Task<string> GetHashForFile(ListedItem fileItem, string nameOfAlg, CancellationToken token, ProgressBar progress)
+        public async Task<string> GetHashForFile(ListedItem fileItem, string nameOfAlg, CancellationToken token, double progress)
```

Comment:
Bring back ProgressBar. Otherwise it is not working.

Reply:
Fixed it by revoking back to progressBar [here](https://github.com/VoidExploiter/files-uwp/commit/72ed0807f351df77a62ea6ef85d12d8be872949a)